tests/Resources/DataFixtures/Page/template_with_xinclude_properties.xml

```diff
@@ -1,12 +1,10 @@
 <?xml version="1.0" ?>
 
-<template xmlns="http://schemas.sulu.io/template/template"
+<properties xmlns="http://schemas.sulu.io/template/template"
```

Comment:
without any namespace it says that `properties` (without namespace) is not expected.